### ISSUE TITLE
extra axis ticks being added

### ISSUE THREAD

**Issue by AlFontal** [has2k1/plotnine#278]
This is a minor but annoying issue. I believe this might be happening due to `matplotlib`'s latest release?
Basically, axis ticks are not just added at the left and bottom sides of the plots but also at the right and top sides.

Running the first example of `plotnine`'s readme:
```{{python}}
import plotnine as p9
from plotnine.data import mtcars

(p9.ggplot(mtcars)
 + p9.aes('wt', 'mpg')
 + geom_point())
```

![image](https://user-images.githubusercontent.com/10958332/58546868-91db6500-8206-11e9-80d0-4bd7051dcf38.png)

**Comment by TyberiusPrime** [comment]
Yes and it makes a lot of plotnine's test cases fail.

Unfortunately, it seems to be more involved than just sticking
```
d['ytick.right'] = False
x['xtick.top'] = False
``` into rcParams.

**Comment by has2k1** [comment]
Yes that is the issue. It is due to the changes introduced in https://github.com/matplotlib/matplotlib/pull/12839. Matplotlib does not handle ticks neatly and we have to work around a few quirks.

**Comment by AlFontal** [comment]
I believe this has already been solved, right?

**Comment by has2k1** [comment]
@AlFontal, it is still an issue.

**Comment by maxgriswold** [comment]
Has this issue been resolved yet? Is there an ad-hoc method to remove while an update is being developed?

**Comment by TyberiusPrime** [comment]
The fix for this was included in the 0.6 release.
